document-portal/document-portal-fuse.c

```diff
@@ -2141,7 +2141,7 @@ forget_one (fuse_ino_t ino,
 {
   g_autoptr(XdpInode) inode = xdp_inode_from_ino (ino);
 
-  g_debug ("FORGET %lx %ld", ino, nlookup);
+  g_debug ("FORGET %" G_GINT64_MODIFIER "x %" G_GSIZE_FORMAT, ino, nlookup);
```

Comment:
`nlookup` here is a long double, so `%ld` was actually correct. I'm reverting this part.

Reply:
Done.

Reply:
I think you mean `unsigned long` (an integer), not `long double` (a floating-point number)?

Strictly speaking it ought to be `%lu` for an `unsigned long`, but `%ld` is close enough.